cluster-up/cluster/kind/podpreset.sh

```diff
@@ -0,0 +1,56 @@
+ #!/usr/bin/env bash
+
+set -e
+
+source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
+
+function podpreset::enable_admission_plugin() {
```

Comment:
I thinks this has to be renamed to "enable_podpreset_admission_plugin" or parameterize same as "validate_admission_plugin_is_enabled"

Reply:
I didnt called it `enable_podpreset_admission_plugin` because it has `podprest::` prefix,
so we have sense what plugin its going to enable, WDYT?

About parameterize it, `podpreset::enable_admission_plugin` implementation is specific for `PodPreset` feature
it patches `kube-apiserver` manifest by adding `PodPreset` to `--enable-admission-plugins` and `settings.k8s.io/v1alpha1=true` to `--runtime-config`
Other admission plugins might require other parameters to be patched.
Unlike `podpreset::validate_admission_plugin_is_enabled` that could check any admission plugin
(because it checks kube-apiserver command state)

Reply:
then what we have to do is to hardcore PodPreset into `podpreset::validate_admission_plugin_is_enabled` this function cannot be generic if it's under "podpreset" or going to the opposite direction and move this functions and a generic version of  `validate_admission_plugin_is_enabled` out of porpreset.

Reply:
I have changed `podpreset::validate_admission_plugin_is_enabled` to have PodPreset hardcoded, 
If we will need to mess with other admission plugins we can parameterize it